### ISSUE TITLE
slider drag jitter problem

### ISSUE THREAD

**Issue by tkokof** [OGRECave/ogre#1061]
there is a slider drag jitter problem when multiple "mouseMoved" event happen between frame, codes from OgreTrays.cpp with desc comments : 

```C++
void Slider::_cursorMoved(const Ogre::Vector2 &cursorPos, float wheelDelta)
{
    if (mDragging)
    {
         // codes here will calculate the offset based on mHandle's member value : mLeft
        Ogre::Vector2 co = Widget::cursorOffset(mHandle, cursorPos);
        Ogre::Real newLeft = mHandle->getLeft() + co.x - mDragOffset;
        Ogre::Real rightBoundary = mTrack->getWidth() - mHandle->getWidth();

        // codes here will set the mHandle's mLeft value, but it could be an "async" operation
        // by set mHandle's mPixelLeft and sync with mLeft in the next update,
        // this means if multiple "mouseMoved" during one frame, mLeft could be incorrect ...
        mHandle->setLeft(Ogre::Math::Clamp<int>((int)newLeft, 0, (int)rightBoundary));
        setValue(getSnappedValue(newLeft / rightBoundary));
    }
}
```

a simple update call seems could fix the slider drag jitter problem, but I'm not so sure about it ...

```C++
void Slider::_cursorMoved(const Ogre::Vector2 &cursorPos, float wheelDelta)
{
    if (mDragging)
    {
        Ogre::Vector2 co = Widget::cursorOffset(mHandle, cursorPos);
        Ogre::Real newLeft = mHandle->getLeft() + co.x - mDragOffset;
        Ogre::Real rightBoundary = mTrack->getWidth() - mHandle->getWidth();

        mHandle->setLeft(Ogre::Math::Clamp<int>((int)newLeft, 0, (int)rightBoundary));
        setValue(getSnappedValue(newLeft / rightBoundary));
        
        // hotfix jitter ?
	mHandle->_update();
    }
}
```

**Comment by paroj** [comment]
does calling `_setLeft` instead also work for you?

**Comment by paroj** [comment]
fixed by c50fa4220c87a8ff1d3be67c53ceb8c7b3624f95

**Comment by tkokof** [comment]
can not find _setLeft method in my code repo ...
btw, TextBox seems also have the slider drag jitter problem since the similar implementation
